docs/programming_guide/experiment_tracking.rst

```diff
@@ -0,0 +1,131 @@
+.. _experiment_tracking:
+
+###################
+Experiment Tracking
+###################
+
+***********************
+Overview and Approaches
+***********************
+
+In a federated computing setting, the data is distributed across multiple devices or systems, and training is run
+on each device independently while preserving each client's data privacy.
+
+Assuming a federated system consisting of one server and many clients and the server coordinating the ML training of clients,
+we can interact with ML experiment tracking tools in two different ways: 
+
+    - Client-side experiment tracking: Each client will directly send the log metrics/parameters to the ML experiment tracking server or local file system
+    - Server-side experiment tracking: Clients will send the log metrics/parameters to FL server, and the FL server will send the metrics to ML experiment
+      tracking server or local file system
+
+Each approach will have its use cases and unique challenges. In NVFLARE, we developed a server-side approach:
```

Comment:
This sentence might give the impression that NVFlare only supports server-side tracking.

We should say NVflare supports both client-side tracking and server-side tracking.

If we add our "Receivers" at the client side we could achieve client-side tracking with local file system.

If they want clients to send directly to some tracking server, they need to set up that tracking server and in their custom code import the corresponding api and call it themselves.

Reply:
I have tested that before.

Only benefit is that you can log/receive cross-formats (for example TB->MLflow)